bciers/apps/administration/tests/components/facilities/FacilityForm.test.tsx

```diff
@@ -99,10 +218,9 @@ describe("FacilitiesForm component", () => {
     const submitButton = screen.getByRole("button", { name: /submit/i });
     expect(submitButton).toBeEnabled();
   });
-
   it("renders the empty LFO facility form when creating a new facility", async () => {
```

Comment:
"Single Facility Operation" isn't an  LFO type, so what I think happened here (haven't tested) is that the input does have the value `Single Facility Operation` (so the test passes), but the value doesn't actually show in the UI because it's not one of the dropdown's allowed options. If we change the mock data here to one of the allowed types the test will be more representative of the actual app.